main.go

```diff
@@ -167,11 +167,12 @@ func main() {
 		DefaultWorkloadClusterNamespaces: defaultNamespaces,
 	}
 
-	eventsLoggerSecret := eventsloggersecret.Reconciler{
-		Client: mgr.GetClient(),
+	eventsLoggerConfig := eventsloggerconfig.Reconciler{
+		Client:                           mgr.GetClient(),
+		DefaultWorkloadClusterNamespaces: defaultNamespaces,
```

Comment:
Why did you add the defaultNamespaces?

Reply:
Right now we get all events

Reply:
That's not what I saw when deleting code from grafana-agent : https://github.com/giantswarm/logging-operator/pull/263/files#diff-3001c64ac5618ddba6d0853ab94895cae21bfb0f529ed778c0ec9814066454acL71

Reply:
Oh right. What do you think about moving the grafana-agent config to a template first? This should be a smaller PR to test right?